storm-server/src/main/java/org/apache/storm/utils/EnumUtil.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+package org.apache.storm.utils;
+
+import java.util.EnumMap;
+import java.util.function.Function;
+
+public class EnumUtil {
+    public static String toString(Enum type) {
```

Comment:
Could you put some documentation on this.  It was not totally clear why this was being called, but now it looks like it is for the metrics name

Reply:
Yes it's a helper for grouped metrics for now. Would you suggest a better naming?

Reply:
You might call it toMetricName instead.